lib/ui/src/modules/ui/containers/stories_panel.js

```diff
@@ -25,11 +33,13 @@ export const mapper = (state, props, { actions }) => {
     sortStoriesByKind
   );
 
-  const storiesHierarchy = createHierarchy(filteredStories);
-  const selectedHierarchy = resolveStoryHierarchy(selectedKind, hierarchySeparator);
+  const storiesHierarchies = createHierarchies(filteredStories);
+
+  const [, storyName] = resolveStoryRoots(selectedKind, rootSeparator);
```

Comment:
I don't understand why exporting the `resolveStoryRoots` ?  You can hide it under the `resolveStoryHierarchy` method.

Reply:
`resolveStoryRoots` is exported for 2 reasons:
1) It's also needed in the stories_panel container.
2) It's has separate unit tests.

Btw I renamed `resolveStoryRoots` to `resolveStoryHierarchyRoots`.